scripts/preconversion_assessment_script.py

```diff
@@ -414,20 +427,26 @@ def main():
 
         # Generate report message and transform the raw data into entries for
         # Insights.
-        output.message = generate_report_message(highest_level)
+        message, alert = generate_report_message(highest_level)
```

Comment:
This could be done by `output.message, output.alert = generate_report_message(highest_level)` if you wanted to save a line or two :-)

Reply:
Sure, thanks! updated :+1: